src/db/db.ts

```diff
@@ -2,6 +2,8 @@ import { Knex } from 'knex';
 
 import { Cursor } from '../types/trigger'
 
+export const defaultTimestampColumn = 'createdAtTime';
```

Comment:
Not sure if we want to rather stick with an enum to allow for future extension? i.e. something like
```javascript
export enum DefaultTimestampColumn {
  createdAtTime = 'createdAtTime'
}
```
Thoughts?

Reply:
not sure exactly what you mean / what you imagine future possible extension would be?

Reply:
I was thinking about `updatedAtTime`, but in the case of an upsert we'd probably want to affect that column. 

If you're good with just a simple `const`, then let's roll on with [YAGNI](https://en.wikipedia.org/wiki/You_aren't_gonna_need_it)!